### ISSUE TITLE
winning icons!!!!

### ISSUE THREAD

**Issue by unicycleclown8** [ShadowMario/FNF-PsychEngine#11562]
### What feature do you want to get added on the **base** engine?

pls

https://user-images.githubusercontent.com/121277045/209275215-bc4ea100-0995-495f-87d3-77549640635b.MOV

### To test your sight, and reliability, please select the option of what should NOT be requested.

Proper credit that was forgotten

**Comment by Gavman69** [comment]
![image](https://user-images.githubusercontent.com/85568813/209277128-766af37f-038a-48d4-af6a-92f858e58b8b.png)

**Comment by Rom4chek** [comment]
💀💀💀💀

**Comment by TheKitBoi** [comment]
there's nothing we can do about it
we have to wait for SM to close or delete this

**Comment by Jevilm** [comment]
honest opinion
stop fucking trolling. you arent fucking funny. all you do is break a fucking rule and expect attention. get a fucking life. 👎

**Comment by PDL2CGamejolt** [comment]
Why do we have to sexualize everything?!? - Emkay, r/TIHI | Damn Tails, you thicc 😳at 4:02

**Comment by PDL2CGamejolt** [comment]
THERES EVEN SOUNDS 💀😬😬😬💀💀💀

**Comment by PDL2CGamejolt** [comment]
BRUH, WHY.

**Comment by ShadowMilky** [comment]
yep i have seen worse

**Comment by IdiotLimitedEdition** [comment]
> yep i have seen worse

You are right this is not even the tip of the iceberg its like the exosphere at best
